src/main/java/io/vertx/starter/config/util/ConfigUtil.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2019 Red Hat, Inc.
```

Comment:
General question, does it make sense to that part of the project under RedHat/Apache IP and the other daniel/Apache in the headers?

Reply:
I'm not the IP expert but I think it's fine if the copyright mentions the owner when the file was first created. Any comment @jponge ?